src/App.js

```diff
@@ -3,20 +3,23 @@ import './App.scss';
 
 import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoLisst/TodoList';
+
+const preparedTodos = todos.map((todo) => {
+  const user = users.find(person => person.id === todo.userId);
+
+  const copy = { ...todo };
+
+  copy.user = user;
+
+  return copy;
+});
```

Comment:
Here is the easier way to perform such operation:
```suggestion
const preparedTodos = todos.map((todo) => ({
  ...todo,
  user: users.find(person => person.id === todo.userId),
}));
```